milli/src/update/facet/bulk.rs

```diff
@@ -171,12 +168,15 @@ impl<R: std::io::Read + std::io::Seek> FacetsUpdateBulkInner<R> {
                 match database.get(wtxn, key)? {
                     Some(prev_value) => {
                         let old_bitmap = &prev_value[1..];
-                        CboRoaringBitmapCodec::merge_into(
-                            &[Cow::Borrowed(value), Cow::Borrowed(old_bitmap)],
-                            &mut buffer,
-                        )?;
+                        CboRoaringBitmapCodec::merge_deladd_into(value, old_bitmap, &mut buffer)?;
```

Comment:
Is it correct to use `&prev_value[1..]`, here?

Reply:
I think so:

1. The DB is `heed::Database<FacetGroupKeyCodec<ByteSliceRefCodec>, FacetGroupValueCodec>`.
2. `FacetGroupValueCodec` contains one byte of size (representing the level I think, starting at 1) followed by a roaring bitmap. So `prev_value[0]` is the size, and `prev_value[1..]` is the bitmap.

This is unchanged from the previous version of the code.

Reply:
Do you want me to add some code comment here?

Reply:
That's OK, I think, thank you. Indeed, a small comment could help a little bit!

Reply:
Done